metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_helper.py

```diff
@@ -17,3 +18,13 @@ def unquote_and_decode_unicode_escape_seq(
     cleaned_string = string.encode().decode("unicode-escape")
 
     return cleaned_string
+
+
+def parse_labels(labels_str: str) -> Dict[str, str]:
+    pattern = r'STRUCT\("([^"]+)", "([^"]+)"\)'
+
+    # Find all matches in the labels string
+    matches = re.findall(pattern, labels_str)
+
+    labels_dict = {key: value for key, value in matches}
+    return labels_dict
```

Comment:
```suggestion
    # Map ?? to ??
    return dict(re.findall(pattern, labels_str))
```
I don't actually know what the keys and values of the returned dict represent. Let's document that somewhere